ctapipe/image/reducer.py

```diff
@@ -131,6 +131,28 @@ class TailCutsDataVolumeReducer(DataVolumeReducer):
         "normal TailcutCleaning is used.",
     ).tag(config=True)
 
+    def __init__(self, subarray, config=None, parent=None, **kwargs):
+        """
+        Parameters
+        ----------
+        subarray: ctapipe.instrument.SubarrayDescription
+            Description of the subarray
+        config: traitlets.loader.Config
+            Configuration specified by config file or cmdline arguments.
+            Used to set traitlet values.
+            Set to None if no configuration to pass.
+        kwargs
+        """
+        super().__init__(config=config, parent=parent, subarray=subarray, **kwargs)
+
+        self.cleaner = TailcutsImageCleaner(parent=self, subarray=self.subarray)
+
+        self.image_extractor = ImageExtractor.from_name(
+            self.image_extractor_type,
+            subarray=self.subarray,
+            config=self.config
```

Comment:
this should be `parent=self`, not `config=self.config` to correctly build the hierarchy.

Here you would give the config for the DataVolumeReducer as the config for the ImageExtractor, which is wrong.

Reply:
Just as above with the cleaner

Reply:
wouldn't `parent=self`  take care of this? Usually you do not have to pass the config manually